ethereum_bridge/src/parameters.rs

```diff
@@ -218,6 +219,31 @@ impl EthereumBridgeConfig {
     }
 }
 
+/// Get the Ethereum address for wNam from storage, if possible
+pub fn read_native_erc20_address<DB, H>(
+    storage: &Storage<DB, H>,
+) -> Result<EthAddress>
+where
+    DB: storage::DB + for<'iter> storage::DBIter<'iter>,
+    H: storage::traits::StorageHasher,
+{
+    let native_erc20 = bridge_storage::native_erc20_key();
+    match storage.read(&native_erc20) {
+        Ok((Some(bytes), _)) => {
+            Ok(EthAddress::try_from_slice(bytes.as_slice()).expect(
+                "Deserializing the Native ERC20 address from storage \
+                 shouldn't fail.",
+            ))
+        }
```

Comment:
call `StorageRead::read` from `core` instead, maybe? it does a borsh deserialize automatically

Reply:
Yeah `StorageRead::read` is much nicer!